cardinal/decorators.py

```diff
@@ -35,3 +35,20 @@ def inner(*args, **kwargs):
         return inner
 
     return wrap
+
+def event(triggers):
```

Comment:
Would you mind writing a test for this decorator similar to the ones already defined in `cardinal/test_decorators.py`? We're not there yet with unit testing, but I've started with the low hanging fruit (decorators and exceptions!) and would like to try to keep the course. :)